.github/workflows/update-docker-hub-readmes.yml

```diff
@@ -0,0 +1,42 @@
+name: Docker CI Ubuntu
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Should we include a `paths` option so we only run this job if the `docs` folder changes? `paths: [ 'docs/**' ]` would probably do it.

It's not a big deal, but with the GHA problems we seem to have, being more selective about when we run jobs might be useful. :)

Reply:
ah good call, i'll add that back in.